### PR TITLE
[FLINK-28315][runtime-web] introduce aggregate stats in tables of the subtasks and taskmanagers

### PR DESCRIPTION
**Pull request by yangjunhan**
## What is the purpose of the change

A subtask for the [FLIP-241](https://cwiki.apache.org/confluence/display/FLINK/FLIP-241%3A+Completed+Jobs+Information+Enhancement), the referenced issue is FLINK-28315.

## Brief change log

Split the per-subtask metrics and aggregated metrics into tabs, and add a table of aggregated metrics.

Extend the original subtasks table by adding extra columns:
- "Accumulated Time"
- "Status Durations" 

Under the TaskManagers tab, each taskManager row supports an action of viewing the "Aggregated Metrics". The modal's content table is identical to that under the subtask tab.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? yes
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by yangjunhan** [comment]
Hi, @simplejason. Could you please help review this PR regarding the runtime-web changes?

File: flink-runtime-web/web-dashboard/src/app/share/customize/duration-badge/duration-badge.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<span nz-tooltip [nzTooltipTitle]="tooltip">
+  {{ duration ? (duration | humanizeDuration) : '-' }}
+</span>
```
Comment (simplejason) [inline flink-runtime-web/web-dashboard/src/app/share/customize/duration-badge/duration-badge.component.html:20]:
```diff
- duration ? (duration | humanizeDuration) : '-'
+ duration | humanizeDuration
```

File: flink-runtime-web/web-dashboard/src/app/share/customize/duration-badge/duration-badge.component.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<ng-template #tooltip>
+  <span>{{ state }}: {{ duration ? (duration | humanizeDuration) : '-' }}</span>
+</ng-template>
```
Comment (simplejason) [inline flink-runtime-web/web-dashboard/src/app/share/customize/duration-badge/duration-badge.component.html:24]:
```diff
- duration ? (duration | humanizeDuration) : '-'
+ duration | humanizeDuration
```

**Review by xintongsong — APPROVED** [review]
Thanks @yangjunhan and @simplejason. Manually verified and LGTM. Merging once the ci succeed.

**Comment by xintongsong** [comment]
@flinkbot run azure

**Pull request closed without merging**